KesaApplication/ViewController.swift

```diff
@@ -10,16 +10,130 @@ import UIKit
 
 class ViewController: UIViewController {
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+    // 4 labels
+    @IBOutlet weak var firstNameLabel: UILabel!
+    @IBOutlet weak var lastNameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet weak var passwordLabel: UILabel!
+    
+    // 4 text fields
+    @IBOutlet weak var firstName: UITextField!
+    @IBOutlet weak var lastName: UITextField!
+    @IBOutlet weak var email: UITextField!
+    @IBOutlet weak var password: UITextField!
+    
+    
+    // When the login button is pressed, ignore all text fields
+    @IBAction func loginButtonPressed(sender: UIButton) {
+        self.firstName.resignFirstResponder() // don't worry about the keyboard
+        self.lastName.resignFirstResponder()
+        self.email.resignFirstResponder()
+        self.password.resignFirstResponder()
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    
+    // When the screen is pressed while editing, hide the keyboard
+    override func touchesBegan(touches: Set<UITouch>, withEvent event: UIEvent?) {
+        self.view.endEditing(true)
```

Comment:
For instance, when the user wants to stop editing one of the text fields then the keyboard is brought up, if the user presses anywhere on the screen except the keyboard, the keyboard is brought down.